### PR TITLE
Add private packages from private channel

### PR DESCRIPTION
**Pull request by AshleyZSY**
*Description of changes:*

Add private packages from private channel. 

Tested by building image and verifeid by conda list

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install pytest-jupyter
+pytest -vv -r ap /opt/conda/lib/python3.10/site-packages/maxdome_jupyter_server_extension || exit $?
```
Comment (claytonparnell) [inline test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh:4]: note - the python version might be updated in v2. Would be good to get the install path/python version dynamically, something like https://github.com/aws/sagemaker-distribution/blob/main/template/v1/Dockerfile#L98

**Review by claytonparnell — CHANGES_REQUESTED** [review]

**Review by nikhilumeshsargur — APPROVED** [review]
approving the changes

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install pytest-jupyter
```
Comment (balajisankar15) [inline test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh:3]: can you move this to the corresponding docker file and use micromamba for installation? Seems like this package is availabe in conda forge

https://anaconda.org/conda-forge/pytest-jupyter

Note: Use --freeze-installed flag when installing from conda forge

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_session_manager_test.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+!/bin/bash
+
+pip install pytest-jupyter
```
Comment (balajisankar15) [inline test/test_artifacts/v2/scripts/run_maxdome_jupyter_session_manager_test.sh:3]: same

**Review by balajisankar15 — CHANGES_REQUESTED** [review]

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh (hunk 2 above)
Comment (AshleyZSY) [reply test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh]: Fixed with follow up commit.

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_session_manager_test.sh (hunk 3 above)
Comment (AshleyZSY) [reply test/test_artifacts/v2/scripts/run_maxdome_jupyter_session_manager_test.sh]: Fixed with follow up commit.

File: test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh (hunk 1 above)
Comment (AshleyZSY) [reply test/test_artifacts/v2/scripts/run_maxdome_jupyter_server_extension_test.sh]: Fixed with follow up commit.

**Review by nikhilumeshsargur — APPROVED** [review]
approving the changes

**Review by claytonparnell — APPROVED** [review]

**Review by balajisankar15 — APPROVED** [review]

**Pull request merged**